src/App.js

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
 
+  const todoList = [
+    { id: 1, title: "Complete assignment" },
+    { id: 2, title: "Task 2" },
+    { id: 3, title: "Task 3" }
+  ];
+
 function App() {
   return (
-    <div style={{ textAlign: 'center' }}>
-      <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
+    <div>
+      <h1>Todo List</h1>
+      <ul>
+        {todoList.map((todo) => (
+        <li key={todo.id}>{todo.title}</li>
```

Comment:
one small detail - because the `<li>` is inside of the perentheses, you'll want to indent this line to show it's a child